### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by JensFastrup**
Proposed readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
 
-'/<username>/follow' 	= follows the user specified and redirects to following users timeline
 
-'/<username>/unfollow'	= unfollows the user specified and redirects to the following users timeline
 
-'/add_message' POST	= POST with a message under the user's tweet
+## Installation on Windows/Linux/macOS
 
-'/login' GET		= login and redirects to user timeline
+Assuming you have Git Bash and Docker. Run the follwing in Git Bash or other preferred shell.
```
Comment (TikoyX) [inline README.md:12]:
```suggestion
You need Docker installed for this project. To install this look at [Docker](https://www.docker.com).
After ensuring Docker is installed clone the repository:
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,42 @@
-# devUps
-For the course DevOps at ITU
+# MiniTwit
 
-# Routing Notes
-'/' 				= renders a user timeline
+MiniTwit is a dashboard/blogging platoform where users can make posts. Originally made in Python, it is currently in
```
Comment (CptF1nn) [inline README.md:3]: typo

File: README.md [hunk 3]
```diff
@@ -1,26 +1,42 @@
...
-'/<username>/unfollow'	= unfollows the user specified and redirects to the following users timeline
 
-'/add_message' POST	= POST with a message under the user's tweet
+## Installation on Windows/Linux/macOS
 
-'/login' GET		= login and redirects to user timeline
+Assuming you have Git Bash and Docker. Run the follwing in Git Bash or other preferred shell. 
 
-'/login' POST		= if succesful, login and redirect to user timeline, otherwise redirect to error
+```Git Bash
```
Comment (TikoyX) [inline README.md:14]:
```suggestion
```shell
```

File: README.md [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
+git clone https://github.com/TikoyX/devUps.git
+```
+From the directory, then run the follwing command with Docker running. 
 
-'/register' GET		= if logged in, goes to timeline
+```Git Bash
```
Comment (TikoyX) [inline README.md:19]:
```suggestion
```shell
```

File: README.md [hunk 5]
```diff
@@ -1,26 +1,42 @@
...
+the application.  
 
-'/register' POST		= validates register form, redirects to login if succesful, and redirects to register if error
+(Should this give problems with prior docker-files try to run "docker-compose build --no-cache")
 
-'/logout'			= Log out user and redirect to public timeline
 
+## Usage
+
+To have an active development environment, we suggest running the frontend directoiry with "npm start" and downloading
```
Comment (TikoyX) [inline README.md:30]:
```suggestion
To have an active development environment, run the following command inside the *minitwit-frontend* directory `npm start`.
```

File: README.md [hunk 6]
```diff
@@ -1,26 +1,42 @@
...
 
-'/register' POST		= validates register form, redirects to login if succesful, and redirects to register if error
+(Should this give problems with prior docker-files try to run "docker-compose build --no-cache")
 
-'/logout'			= Log out user and redirect to public timeline
 
+## Usage
+
+To have an active development environment, we suggest running the frontend directoiry with "npm start" and downloading 
+the proper extensions for any development.
```
Comment (TikoyX) [inline README.md:31]:
```suggestion

```

File: README.md [hunk 7]
```diff
@@ -1,26 +1,42 @@
...
-'/logout'			= Log out user and redirect to public timeline
 
+## Usage
+
+To have an active development environment, we suggest running the frontend directoiry with "npm start" and downloading 
+the proper extensions for any development. 
+
+## Contributing
+
+For contribution guidelines, see contributing.md.
```
Comment (TikoyX) [inline README.md:35]:
```suggestion
For contribution guidelines, see [contributing.md](contributing.md). 
```

File: README.md (hunk 2 above)
Comment (TikoyX) [inline README.md:3]:
```suggestion
MiniTwit is a dashboard/blogging platform where users can make posts. Originally made in Python with Flask, it is currently in 
```

**Review by TikoyX — CHANGES_REQUESTED** [review]

**Comment by JensFastrup** [comment]
ready for review

**Review by TikoyX — APPROVED** [review]

**Pull request merged**